recipes/openms/meta.yaml

```diff
@@ -22,16 +23,17 @@ requirements:
     - glib
     - autoconf
     - automake
+    - doxygen
```

Comment:
I thought it would be nice for a standalone installation if it fits the build time. On the other hand I agree there's probably not a real use case - I'll remove it with the next commit.